src/components/NegotiationPosts.vue

```diff
@@ -142,12 +145,15 @@ export default {
       }
     },
     async addMessage() {
+      console.log(this.scope.toUpperCase())
       // send a message and add the newly created post
       await this.addMessageToNegotiation({
         data: {
-          resourceId: this.negotiation.requests[0].resources[0].id,
+          resourceId: this.resourceId, //this.negotiation.requests[0].resources[0].id,
```

Comment:
if the comment is not necessary, please remove it

Reply:
Fixed in 338765dbdacd1bacd4779ccc2b2d75595643e489